zynq_boot_bin/build_boot_bin.ps1

```diff
@@ -45,16 +46,16 @@ Remove-Item -Recurse -Force -ErrorAction:SilentlyContinue $OUTPUT_DIR
 mkdir -p $OUTPUT_DIR
 mkdir -p $BUILD_DIR
 
-cp $HDF_FILE $BUILD_DIR/
+cp $XSA_FILE $BUILD_DIR/
 cp $UBOOT_FILE $OUTPUT_DIR/u-boot.elf
-cp $HDF_FILE $OUTPUT_DIR/
+cp $XSA_FILE $OUTPUT_DIR/
 
-$BASE_HDF=(Get-Item $HDF_FILE).name
+$BASE_XSA=(Get-Item $XSA_FILE).name
 
-echo "hsi open_hw_design $BASE_HDF" | out-file -encoding ASCII $BUILD_DIR/create_fsbl_project.tcl
+echo "hsi open_hw_design $BASE_XSA" | out-file -encoding ASCII $BUILD_DIR/create_fsbl_project.tcl
 echo 'set cpu_name [lindex [hsi get_cells -filter {IP_TYPE==PROCESSOR}] 0]' | out-file  -Append -encoding ASCII $BUILD_DIR/create_fsbl_project.tcl
 echo 'sdk setws ./build/sdk'  | out-file  -Append -encoding ASCII $BUILD_DIR/create_fsbl_project.tcl
```

Comment:
I think this part of the code works only for HDF case. In newer Xilinx versions there isn't any 'sdk' command, but instead 'platform' needs to be used. 
See an example here: [https://bitbucket.analog.com/projects/SDG/repos/ghdl/browse/bin/create_fsbl_project.tcl#23](https://bitbucket.analog.com/projects/SDG/repos/ghdl/browse/bin/create_fsbl_project.tcl#23 )